### PR TITLE
article-images

### PR DESCRIPTION
**Pull request by diana-lakatos**

### CODE REVIEW

File: src/docs/04-articles/03-docskit-2023-summary.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+   * Feature case studies: Illustrating how specific DocsKit features address common challenges, complete with practical examples and code snippets.
+   * Site case studies: Showcasing fully implemented DocsKit sites. 
+
+* **University course: **We are currently working on a course curriculum for the University of Szeged, which will cover the entire product development life cycle for a PaaS application. platformOS serves as the main example, with DocsKit illustrating documentation, accessibility, and sustainability aspects. 
+
+* **Docs as Code Fundamentals course: **Our [Docs as Code Fundamentals](https://docskit.platformos.com/docs-as-code-course/) course, starting in January 2024, provides a comprehensive understanding of Docs as Code. Ideal for beginners and those refining their skills, the course covers the basics to the editorial workflow. [Sign up for the course](https://docskit.platformos.com/user/register) to access videos for each lesson and participate in interactive webinars for deeper engagement with the instructor and our team.
+
+<a href="/user/register"><img src="./assets/course-registration-banner" alt="Register for the Docs as Code Fundamentals Course"></a>
```
Comment (nemvalid) [inline src/docs/04-articles/03-docskit-2023-summary.mdx:64]:
```suggestion
<a href="/user/register"><img src="./assets/course-registration-banner" alt="Register for the Docs as Code Fundamentals Course" /></a>
```

File: src/docs/04-articles/03-docskit-2023-summary.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+# Partnerships 
+
+**DocsKit is now accessible for any platformOS Partner**. Furthermore, we are actively collaborating to establish a **network of specialized partners** focused on DocsKit development for documentation sites. This initiative aims to provide clients with the option to choose a partner with expertise in DocsKit when they decide to create a DocsKit site. 
+
+For clients interested in directly working with the DocsKit team, we've initiated **discovery calls** where we introduce key features and benefits, compare DocsKit with open-source and SaaS solutions, discuss integrations, showcase example DocsKit powered sites, and outline pricing.
+
+<a href="https://calendar.thetrackapp.com/colin-frost/5939f8d5-c49e-4626-9e70-3f51aa599dcd"><img src="./assets/discovery-call-banner" alt="Schedule a 20-minute discovery call"></a>
```
Comment (nemvalid) [inline src/docs/04-articles/03-docskit-2023-summary.mdx:72]:
```suggestion
<a href="https://calendar.thetrackapp.com/colin-frost/5939f8d5-c49e-4626-9e70-3f51aa599dcd"><img src="./assets/discovery-call-banner" alt="Schedule a 20-minute discovery call" /></a>
```

**Comment by nemvalid** [comment]
@BratOBlog could you please take a look at https://github.com/Platform-OS/docskit-landing/pull/73/files#diff-82740d853365ce999afdf38e80c983d464883f379456690cc3d2226f0dfca156R64 and https://github.com/Platform-OS/docskit-landing/pull/73/files#diff-82740d853365ce999afdf38e80c983d464883f379456690cc3d2226f0dfca156R72 on this branch (`2023-summary-article`) in `/src/docs/04-articles/03-docskit-2023-summary.mdx`? 
Please fix the image urls by importing them and passing the imported value to the `img` tag's src attribute? 
something like 
```mdx
import registrationBanner from './assets/course-registration-banner.png';

<a href="/user/register">
  <img src={registrationBanner} alt="Register for the Docs as Code Fundamentals Course" />
</a>
```
and the same for the other one.

**Comment by diana-lakatos** [comment]
Thank you @BratOBlog and @nemvalid! 
The banners look and work well now! There is one more image that is not shown, the `customization.gif`.  I included it the way I include simple images, should I change this? Thanks!

**Comment by BratOBlog** [comment]
@diana-lakatos You're Welcome! I have made an alternate quick fix, hope it is fine.

**Comment by diana-lakatos** [comment]
@nemvalid @BratOBlog I made some Markdown fixes, and checked the preview, I think this is now ready to go to staging. Thank you!

**Review by nemvalid — APPROVED** [review]
👍

**Pull request merged**